charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-ui.yaml

```diff
@@ -63,7 +66,11 @@ data:
         proxy_cache_bypass  $http_upgrade;
         proxy_set_header Upgrade           $http_upgrade;
         proxy_set_header Connection        "upgrade";
+{{- if .Values.istio.enabled }}
+        proxy_set_header Host              "pxcentral-backend";
+{{- else }}
```

Comment:
Why was this needed?

Reply:
istio has specific requirements on the network calls to happen. so handling it specifically for istio , for normal installation i am not going with this flow , as it can bring in multiple hops in the network calls